AWS Project/about.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ABOUT US</title>
+</head>
+<body>
+    <h1>Samanth/Pranav edit this page</h1>
```

Comment:
Please get the latest stuff from pranav and sam and add here. I dont think they know git so we cant ask them to add here